packages/battery_plus/macos/battery_plus.podspec

```diff
@@ -6,8 +6,7 @@ Pod::Spec.new do |s|
   s.version          = '0.0.1'
   s.summary          = 'No-op implementation of the macos battery_plus to avoid build issues on macos'
   s.description      = <<-DESC
-  No-op implementation of the battery_plus plugin to avoid build issues on macos.
-  https://github.com/flutter/flutter/issues/46618
```

Comment:
It would be actually nice to keep the reference/explanation here that this is just a dummy file required as a workaround, because these are creating confusion: https://github.com/fluttercommunity/plus_plugins/issues/22